modules/howtos/pages/full-text-searching-with-sdk.adoc

```diff
@@ -10,20 +10,23 @@ FTS allows you to create, manage, and query full-text indexes on JSON documents
 
 It uses natural language processing for querying documents, provides relevance scoring on the results of your queries, and has fast indexes for querying a wide range of possible text searches.
 
-Supported query types include simple queries like Match and Term queries; range queries like Date Range and Numeric Range; and compound queries for conjunctions, disjunctions, and/or boolean queries.
+Search queries are executed at the cluster level and include simple text queries like Match and Term queries; range queries like Date Range and Numeric Range; and compound queries for conjunctions, disjunctions, and/or boolean queries.
 
 The Node.js SDK exposes an API for performing FTS queries which abstracts some of the complexity of using the underlying REST API.
 
+== Examples
 
-// As of Couchbase Server 6.5, FTS...
+Examples on this page are not copy pastable without having the related xref:6.5@server:manage:manage-settings/install-sample-buckets.adoc[Sample Buckets] installed and FTS indexes created. 
 
-== Examples
+We use the `travel-sample` dataset and `index-hotel-description` FTS index for all but one example. 
+
+The `beer-sample` dataset and `index-brewery-by-daterange` FTS index are used only for the Date Range example.
 
-Search queries are executed at the cluster level (not bucket or collection). All examples below will console log our returned documents along with their metadata and rows, each returned document has an index, id, score and sort value.
+xref:full-text-searching-index-creation.adoc[Setting up the FTS indexes]
```

Comment:
We really need a programmatic method of creating the index with the SDK, as this is an SDK howto:

### == Search Index Creation

Use `UpsertIndex()` to create a Search Index for each sample bucket:

[source,golang]
----
```
mgr.UpsertIndex(SearchIndex{
		Name:       "travelsample-hotel-description",
		Type:       "fulltext-index",
		SourceType: "couchbase",
		SourceName: globalBucket.Name(),
	}

```----

Now create the Search Index for the _Beer Sample_ in the same way:

[source,golang]
----
```
mgr.UpsertIndex(SearchIndex{
		Name:       "index-brewery-by-date-range",
		Type:       "fulltext-index",
		SourceType: "couchbase",
		SourceName: globalBucket.Name(),
	}
```
----